docs/application/native/guides/connectivity/nfc.md

```diff
@@ -100,14 +100,13 @@ The main features of the NFC API include:
 
   > **Note**
   >
-  > Pay attention to the following:
-  > - HCE operation requires launching HCE application in Tizen Application Framework. Because of this, enough time is required for the data to be transferred to the application.
-  > - If you consider time efficient, we recommend that you don't create HCE applications and communicate with the Secure Elements provided in the device.
-  > - Alternatively, make sure that the HCE application is always in the launching state.
+  > - HCE operations begin with the launch of HCE application in Tizen Application Framework. With the execution of these operations, the data transfer to the application needs more time.
+  > - In case, you consider time efficiency, it is recommend not to create HCE applications and communicate with the Secure Elements provided in the device.
+  > - Alternatively, ensure that the HCE application is always in the launch state.
 
 ## Prerequisites
 
-To enable your application to use the NFC functionality:
+To enable use of NFC functionality in your application:
```

Comment:
- Enable use of NFC functionality in your application:

Reply:
Hello, @nawab-samsung 
This "Prerequisites" section is used to give the **precondition** to use the function.
If you check other pages also, you'd see they also have "Prerequisites" section to describe the things what to do **before** you try to implement the function.

Your suggestion means a user would enable the NFC function by below steps,
so I don't thing it is correct sentence in here.

Also, as we have lots of guide pages,
please check the consistency with other pages.
We have several members in SRI-B reviewer team, and each member tries to change the sentence as their styles.
It is causing various  styles in the document format, and I think following one fixed style is better than that.

Reply:
@mijin85cho  
Thanks for the clarity, I do understand that "to do X one must ensure that certain activities are done.
I have written the sentence in the context that Enabling NFC is one of the prerequisites.

Reply:
# Prerequisites

- You must enable NFC functionality in your application.

Reply:
> I have written the sentence in the context that Enabling NFC is one of the prerequisites.

@nawab-samsung Your newly suggested sentence is totally reversing the order.
The list items  in this section is not based on "Enabling NFC is one of the prerequisites".
Please check that this section is "Prerequisites", which means these **should be prepared to enable the NFC**.
To enable the NFC, actions described in this section must be set, otherwise you **can't** enable the NFC. 

To enable the NFC, 
   1. you must add the `http://tizen.org/privilege/nfc` privileges to the `tizen-manifest.xml`
   2. you must add `<nfc.h>` header
   3. Support to the NFC should be checked.... and so on.

It is not the "enable the NFC functionality and ensure."
"To enable NFC, ensure" is the meaning of this section.